crates/svm-codec/src/api/json/receipt.rs

```diff
@@ -0,0 +1,353 @@
+use serde_json::{json, Value};
+
+use crate::api::json::{self, JsonError};
+use crate::api::raw;
+
+use svm_types::receipt::{
+    ExecReceipt, Log, ReceiptError, ReceiptOwned, SpawnAppReceipt, TemplateReceipt,
+};
+
+pub fn decode_receipt(json: &Value) -> Result<Value, JsonError> {
```

Comment:
Same here; you're receiving a raw version, decode it to native (`raw::decode_receipt`), and then encode it to JSON. If so, the function should be called `encode_receipt`.